js/components/harvest/job.vue

```diff
@@ -60,7 +57,6 @@ import {
 } from 'models/harvest/job';
 import {STATUS_CLASSES, STATUS_I18N} from 'models/harvest/item';
 import {PageList} from 'models/base';
-import $ from 'jquery';
```

Comment:
There will be a lot of resurrected kittens until this evening :wink:  :tada: :confetti_ball: